process/smartContract/builtInFunctions/esdtLocalBurn.go

```diff
@@ -93,11 +93,7 @@ func (e *esdtLocalBurn) IsInterfaceNil() bool {
 	return e == nil
 }
 
-func checkInputArgumentsForLocalAction(
-	acntSnd, _ state.UserAccountHandler,
-	vmInput *vmcommon.ContractCallInput,
-	funcGasCost uint64,
-) error {
+func checkBasicESDTArguments(vmInput *vmcommon.ContractCallInput) error {
```

Comment:
You should add also this check I think, as you have removed it from the old code and it is not added here:
if vmInput.CallValue.Cmp(zero) != 0 {	
		return nil, process.ErrBuiltInFunctionCalledWithValue	
	}

Reply:
it is there lines 100-102